metaflow/plugins/kfp/kfp.py

```diff
@@ -621,6 +626,14 @@ def _set_container_resources(
             container_op.add_affinity(affinity)
             container_op.add_toleration(toleration)
 
+    # used by the workflow_uid_op and the s3_sensor_op to tighten resources
+    # to ensure customers don't bear unnecesarily large costs
+    def _set_hardcoded_container_resources(self, container_op: ContainerOp):
+        container_op.container.set_cpu_request("0.1")
+        container_op.container.set_cpu_limit("0.5")
+        container_op.container.set_memory_request("10M")
+        container_op.container.set_memory_limit("200M")
```

Comment:
`_set_minimal_container_resources`
Why isn't this also used for `_create_volume`

Reply:
It looks like for `_create_volume`, which creates a [ResourceOp](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ResourceOp), there isn't any way set container resources. `ResourceOp` doesn't even have a `.container` property.